### DISCUSSION TITLE
My fork issues notifications are delivered to the original repo only

### DISCUSSION THREAD

**Discussion by egzumer** [community/community#69294] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

This is the fork
https://github.com/egzumer/uv-k5-firmware-custom

@OneOfElven gets all the issues notifications from my fork including about adding new ones, I don't get any unless I subscribe to any of them manually. I can't find a solution to that. What is going on, looks like a bug.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by krushnarout** [comment]
You should check your notification settings on your forked repository. Make sure that you are set to receive notifications for issues and other activities on your fork.

**Comment by egzumer** [comment]
I've checked the settings multiple times, the only settings for notifications are email notifications for push to repository events. Where else should I look?

![Screen Shot 10-06-23 at 08 57 PM](https://github.com/community/community/assets/14902414/908e2e94-fa40-47d8-ac4e-a312e724e8e0)

**Comment by krushnarout** [comment]
When you fork a repository, GitHub sets it up as a separate entity linked to the original repository, but they share the same issues tracker. This means that any issues, whether new or existing, on your forked repository are essentially managed within the original repository's issues system. As a result, notifications for these issues are sent to the original repository's owner, @OneOfElven, and not to you, unless you manually subscribe to them. To ensure you receive notifications for issues on your forked repository, you'll need to subscribe to them individually. This can be done by clicking on the "Subscribe" button for each issue. Unfortunately, there isn't an automatic notification system in place for forked repositories in this scenario, so you have to manage your subscriptions manually if you want to stay updated on the issues raised in your forked repository. It's important to note that this behavior is not a bug, but rather a feature of how GitHub handles forked repositories and their associated notifications.

**Comment by egzumer** [comment]
I understand, thank you for your response.
This rather unexpected behaviour for me, what if want it to be my separate project based on some other project and still be able to pull changase from the base project? Should I copy it outside of the github and push it to newly created project on my account? I feel like there at least should be an option to make the issue tracking separate.

**Comment by krushnarout** [comment]
If you want your forked repository to function as a separate project with its own independent issue tracking, you would indeed need to copy it outside of GitHub and create a new repository under your account. This way, you can have complete control over the project, including the issue tracking system. However, keep in mind that if you copy the repository and start a new project, you won't be able to pull changes directly from the original project in an automated way like you would with a fork. Instead, you would need to manually manage any updates or changes from the original project that you want to incorporate into your separate project. It's a trade-off between independence and convenience, and GitHub's design currently ties certain functionalities, like issue tracking, to the parent repository for forks.

**Comment by egzumer** [comment]
Well I had a second thought. This is my fork, I implement new functions to it that may cause some bugs, the resulting issues would be related exclusively to me and my fork. Only me can turn on and off the issues section on my fork. How does it make any sense that I don't get any notifications and the upstream repository creator does? And as far as I know the upstream repo has no option to disable notifications from my fork.

Moreover the upstream repo was made hidden for a day which resulted in my repo being reattached to some other upstream repo, which I didn't like so I detached my fork completely. I'm still not getting any notifications about new issues and discussions. Wonder who is getting them now :)

**Comment by egzumer** [comment, marked as answer]
In case anybody else stumbles on this topic, I just discovered, the notification setting are under https://github.com/watching
